cmd/exchange_template/exchange_template_test.go

```diff
@@ -65,18 +68,14 @@ func TestNewExchangeAndSaveConfig(t *testing.T) {
 			WS:   true,
 		},
 	)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 
 	cfgData, err := os.ReadFile(exchangeConfigPath)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if err = saveConfig(testExchangeDir, cfg, exchCfg); err != nil {
-		t.Error(err)
-	}
-	if err = os.WriteFile(exchangeConfigPath, cfgData, file.DefaultPermissionOctal); err != nil {
-		t.Error(err)
-	}
+	require.NoError(t, err, "os.ReadFile must not error")
+
+	err = saveConfig(testExchangeDir, cfg, exchCfg)
+	require.NoError(t, err, "saveConfig must not error")
+
+	err = os.WriteFile(exchangeConfigPath, cfgData, file.DefaultPermissionOctal)
+	assert.NoError(t, err, "os.WriteFile must not error")
```

Comment:
🚧 I asked for this to be `require` 🙂 
You can choose to keep `assert`, but if you do, you __must not_ use the wording `must not` in the message or you'll fail the CI step which ensures assert+should or require+must.
So this is now a required change - pick either, but fix the CI fail.